src/network/netdev/tunnel.c

```diff
@@ -613,13 +640,49 @@ int config_parse_encap_limit(const char* unit,
         return 0;
 }
 
+int config_parse_tunnel_fou_port(const char *unit,
+                                 const char *filename,
+                                 unsigned line,
+                                 const char *section,
+                                 unsigned section_line,
+                                 const char *lvalue,
+                                 int ltype,
+                                 const char *rvalue,
+                                 void *data,
+                                 void *userdata) {
+        Tunnel *t = userdata;
+        uint16_t port = 0;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+        assert(data);
+
+        if (!streq(rvalue, "auto")) {
```

Comment:
Taking 'auto' is necessary? When 'auto' is specified, then the port variable is set to 0. But this is the default value. So, we do not need to set 'auto' to SourcePort=...
I think config_parse_ip_port() is sufficient for these settings.

Reply:
yes